de/01-introduction/01-chapter1.markdown

```diff
@@ -317,17 +317,17 @@ Alternativ kann man Git auch über Homebrew (`http://brew.sh/`) installieren. Ha
 
 <!--Installing Git on Windows is very easy. The msysGit project has one of the easier installation procedures. Simply download the installer exe file from the GitHub page, and run it:-->
 
-Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lade einfach das Installationsprogramm für Windows von der GitHub Webseite herunter und führe es aus:
+Das msysGit Projekt macht die Installation von Git unter Windows sehr einfach. Lad einfach das Installationsprogramm für Windows von der GitHub-Webseite herunter und führe es aus:
```

Comment:
This imperative has incorrect grammar in German. Verben mit Verbstammendung: `-t`, `-d`, `-ig`, `-er`, `-el` bekommen im Imperativ die Endung `-e`.